_episodes/14-nfcore.md

```diff
@@ -359,13 +297,14 @@ nf-core/tools version 2.1
 ┏━━━━━━━━━━━━━━━━━━━┳━━━━━━━┳━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━┓
 ┃ Pipeline Name     ┃ Stars ┃ Latest Release ┃      Released ┃    Last Pulled ┃ Have latest release? ┃
 ┡━━━━━━━━━━━━━━━━━━━╇━━━━━━━╇━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━┩
-│ rnaseq            │   304 │            3.0 │  3 months ago │ 1 minutes ago │ Yes (v3.0)           │
+│ rnaseq            │   394 │            3.4 │  4 weeks ago  │ 1 minutes ago  │ Yes (v3.4)           │
 [..truncated..]
 ~~~
 {: .output}
 
-If not specified, Nextflow will fetch the default branch. For pipelines with a stable release this the default branch is `master` - this branch contains code from the latest release. For pipelines in early development that don't have any releases, the default branch is `dev`.
-
+> ## Development Releases
```

Comment:
I'm wondering if this is necessary. The default behaviour is usually what one wants and it's rarely necessary to know about other branches, unless contributing to development.